presign/base/models.py

```diff
@@ -457,6 +476,25 @@ def send_change_state_email(self, request, action):
         msg.send()
 
 
+class ParticipantCustomMessage(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    participant = models.ForeignKey(Participant, on_delete=models.CASCADE)
```

Comment:
```suggestion
    participant = models.ForeignKey(Participant, on_delete=models.CASCADE, related_name="messages")
```

so we can use `participant.messages` instead of the `get_messages` function (also enables us to have pre-fetching in the future iirc)